main.js

```diff
@@ -0,0 +1,79 @@
+const apiKey = "ee02afd279ba852f5518dbe94195f595";
+
+const searchButton = document.querySelector(".btn-primary");
+const cityInput = document.querySelector('.form-control');
+const weatherContainer = document.getElementById('weather-container');
+const currentWeatherDiv = document.querySelector('.current-weather');
+const forecastDiv = document.querySelector('.forecast');
+
+currentWeatherDiv.style.display = 'none';
+forecastDiv.style.display = 'none';
+
+searchButton.addEventListener('click', () => {
+  const city = cityInput.value;
+  if (!city) {
+   alert('No city name given');
+   return;
+  }
+  fetchCurrentWeather(city);
+  fetchForecast(city);
+});
+
+function fetchCurrentWeather(city) {
+  const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=imperial&appid=${apiKey}`;
+  fetch(url)
+    .then((response) => response.json())
+    .then(function (data) {
+      const cityNameElement = document.getElementById('city-name');
+      const weatherIconElement = document.getElementById('current-weather-icon');
+      const tempElement = document.getElementById('current-temp');
+
+      cityNameElement.textContent = data.name;
+      weatherIconElement.src = `https://openweathermap.org/img/wn/${data.weather[0].icon}@2x.png`;
+      weatherIconElement.alt = data.weather[0].description;
+      tempElement.textContent = `${data.main.temp} °F - ${data.weather[0].description}`;
+
+      currentWeatherDiv.style.display = 'block';
+    })
+    .catch((error) => {
+      console.error('Error, didn\'t fetch weather', error);
```

Comment:
this gives no feedback to the user

Reply:
What would be the corrected code to give feedback to the user?

Reply:
ideally would be an actual message that is rendered in the UI but for now alert could work as well